data/weapons/weapon_data.csv

```diff
@@ -1,59 +1,59 @@
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-Grave LAC,diableavionics_graveac,0,,600,500,,50,,10,30,6,9,1.666,3,KINETIC,50,,0,0.4,,,5,10,0.5,3,,900,,,100,,,,"kinetic8, SR, diable_weapon_bp",,Diable Avionics,,Anti Shield,,,,,,,,,,
+Grave LAC,diableavionics_graveac,0,,600,500,,70,,10,30,6,9,1.666,3,KINETIC,63,,0,0.4,,,5,10,0.5,3,,900,,,100,,,,"kinetic8, SR, diable_weapon_bp",,Diable Avionics,,Anti Shield,,,,,,,,,,
 Grave HMC,diableavionics_gravemg,0,,1150,650,,65,,10,20,11,,,,KINETIC,52,,0,0.2,1,,1,10,0.5,3,,700,,,100,,,,"kinetic15, SR, diable_weapon_bp",,Diable Avionics,,Anti Shield,,,,,,,,,,
 Glowtusk Linear Rifle,diableavionics_glowtusk,0,,1800,600,,500,,50,18,10,,,,HIGH_EXPLOSIVE,400,,0,2.5,,,3,10,3,1,,750,,,350,,,,"he15, SR, diable_weapon_bp",,Diable Avionics,,Anti Armor,,,,,,,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 Mostro Cannon Array,diableavionics_mostro,2,,3250,800,,250,,25,10,20,,,,HIGH_EXPLOSIVE,250,,0,1,,,2,6,0.5,4,,650,,,60,,,,"he17, diable_weapon_bp, rare_bp",,Diable Avionics,,Anti Armor,,,,,,,,,,
-Opfer GV Suppression Artillery,diableavionics_opfergv,3,0.5,8000,900,,10,,20,12,20,320,25,160,KINETIC,10,,0.5,0.75,9999,0.075,2,10,0.05,4,,800,,,25,,,,"kinetic18, diable_weapon_bp, rare_bp",,Diable Avionics,,Shield Suppression,,,,,,,,,,
+Opfer GV Suppression Artillery,diableavionics_opfergv,3,0.5,8000,900,,10,,20,12,20,320,25,160,KINETIC,9,,0.5,0.75,9999,0.075,2,10,0.05,4,,800,,,25,,,,"kinetic18, diable_weapon_bp, rare_bp",,Diable Avionics,,Shield Suppression,,,,,,,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 Winee EMPR,diableavionics_winee,2,0.5,600,600,,25,400,25,50,7,,,,ENERGY,50,,0.5,7.2,5,0.075,,,,,,1000,,,25,,,FIRE_WHEN_INEFFICIENT,"energy5, diable_weapon_bp, rare_bp",,Diable Avionics,,Support,,,,,"%s: Hits on shields while at high hard flux or on hull have a %s chance to arc to weapons and engines, dealing extra EMP damage. ",EMP Arcs|25%,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-Burchel Beam Repeater,diableavionics_burchel,1,0.5,600,600,250,,,,75,8,6,0.5,3,ENERGY,,200,,,0.48,0.02,,,,,1000000,,,,,,,"PD,ANTI_FTR","pd9,beam8, diable_weapon_bp, rare_bp",,Diable Avionics,,Point Defense,,,,,,,,,,
-Glaux Lance Repeater,diableavionics_glaux,2,0.5,2250,700,687,,,,16,11,6,0.605,2,ENERGY,,756,,,0.48,0.02,,,,,1000000,,,,,,,,"energy14, beam10, diable_weapon_bp, rare_bp",,Diable Avionics,,Strike,,,,,,,,,,
+Burchel Beam Repeater,diableavionics_burchel,1,0.5,600,600,300,,,,75,8,12,1,3,ENERGY,,240,,,0.48,0.02,,,,,1000000,,,,,,,"PD,ANTI_FTR,PD_ALSO","pd9,beam8, diable_weapon_bp, rare_bp",,Diable Avionics,,Point Defense,,,,,,,,,,
```

Comment:
Burchel beam repeater damage increased by ~20%, ammo capacity increased to 12 (from 6), ammo regen doubled. flux per second increased to 240 (from 200)